UPGRADING.md

```diff
@@ -4,6 +4,16 @@ Below you'll find special instructions to consider when upgrading to a new versi
 Check out the [change log](CHANGES.md) for a full list of new features and fixes.
 
 
+## To version 3.0.x
+
+Administrators should be aware of the following changes when upgrading to version 3.0.0 or later.
+ecAudit can be upgraded to 3.0.x from version 2.x.x as long as these and any intermediate upgrade instructions are observed.
```

Comment:
This was a bit hard to understand, maybe something like this:
"as long as these and other intermediate upgrade instructions (described below) are followed.

Reply:
Done